lib/erl_interface/doc/src/erl_call_cmd.xml

```diff
@@ -116,6 +116,34 @@
               expressions and returns the result from the last expression.
               Returns <c>{ok,Result}</c> on success.</p>
           </item>
+          <tag><c>-fetch_stdout</c></tag>
+          <item>
+            <p>
+              (<em>Optional.</em>) Executes the code, specified with
```

Comment:
Should be a note that this option does not work with Erlang nodes before OTP-24.

Reply:
You are right! Fixed!